api/api/serializers/docs.py

```diff
@@ -0,0 +1,84 @@
+from django.conf import settings
+
+from api.constants.parameters import TAG
+
+
+CREATOR_COLLECTIONS_DISABLED = """
+Search by creator only. Cannot be used with `q`. The search
+is fuzzy, so `creator=john` will match any value that includes the
+word `john`. If the value contains space, items that contain any of
+the words in the value will match. To search for several values,
+join them with a comma."""
+
+CREATOR = """
+_When `q` parameter is present, `creator` parameter is ignored._
+
+**Creator collection**
+When used with `collection=creator`, returns the collection of media
+by the specified creator. Notice that a single creator's media items
+can be found on several sources, but this collection only returns the
+items from the specified source.
+This is why for this collection, both the creator and the source
+parameters are required, and matched exactly. For a fuzzy creator search,
+use the default search without the `collection` parameter.
+
+**Creator search**
+When used without the `collection` parameter, will search in the creator field only.
+The search is fuzzy, so `creator=john` will match any value that includes the
+word `john`. If the value contains space, items that contain any of
+the words in the value will match. To search for several values,
+join them with a comma.
+"""
+
+CREATOR_HELP_TEXT = (
+    CREATOR if settings.ENABLE_COLLECTIONS else CREATOR_COLLECTIONS_DISABLED
+)
+COLLECTION_HELP_TEXT = f"""
+The kind of media collection to return.
+
+Should be used with `{TAG}`, `source` or `creator`+`source`"""
+
+EXCLUDED_SOURCE_HELP_TEXT = """
+A comma separated list of data sources to exclude from the search.
+Valid values are `source_name`s from the stats endpoint: %(origin)s/v1/%(media_path)s/stats/.
+"""
+SOURCE_HELP_TEXT_COLLECTIONS_DISABLED = """
+A comma separated list of data sources; valid values are
+``source_name``s from the stats endpoint: %(origin)s/v1/%(media_path)s/stats/."""
```

Comment:
Are the double backticks necessary here?
```suggestion
`source_name`s from the stats endpoint: %(origin)s/v1/%(media_path)s/stats/."""
```

Reply:
I copy-pasted if from the old docs, and I also don't know why there are double backticks there.